package.json

```diff
@@ -10,16 +10,16 @@
     "react-scripts": "3.2.0"
   },
   "scripts": {
+    "test" : "eslint *.js src/*",
     "start": "rescripts start",
     "build": "rescripts build",
-    "test": "rescripts test",
     "eject": "react-scripts eject",
     "dev": "concurrently \"BROWSER=none npm start\" \"wait-on http://localhost:3000 && electron .\""
   },
   "rescripts": [
     "env"
   ],
-  "homepage": "./",
+  "homepage": "http://./",
```

Comment:
why did you change this?

Reply:
no clue how that changed, changing back to former